libs/filabridge/src/UibGenerator.cpp

```diff
@@ -43,7 +43,8 @@ UniformInterfaceBlock const& UibGenerator::getPerViewUib() noexcept  {
             .add("viewFromClipMatrix",      1, UniformInterfaceBlock::Type::MAT4, Precision::HIGH)
             .add("clipFromWorldMatrix",     1, UniformInterfaceBlock::Type::MAT4, Precision::HIGH)
             .add("worldFromClipMatrix",     1, UniformInterfaceBlock::Type::MAT4, Precision::HIGH)
-            .add("lightFromWorldMatrix",    1, UniformInterfaceBlock::Type::MAT4, Precision::HIGH)
+            .add("lightFromWorldMatrix",    CONFIG_MAX_SHADOW_CASCADES, UniformInterfaceBlock::Type::MAT4, Precision::HIGH)
```

Comment:
I would hardcode 4 here, and assert that `CONFIG_MAX_SHADOW_CASCADES` is 4 because changing it would break materials.